apps/devhub/tests/test_views.py

```diff
@@ -1141,12 +1159,17 @@ def test_step1_submit(self):
 
     def test_read_dev_agreement_set(self):
         """Store current date when the user agrees with the user agreement."""
-        response = self.client.get(reverse('devhub.submit.1'))
-        previous = response.context['user'].read_dev_agreement
+        self.user.update(read_dev_agreement=None)
 
         response = self.client.post(reverse('devhub.submit.1'), follow=True)
         user = response.context['user']
-        assert user.read_dev_agreement != previous
+        self.assertCloseToNow(user.read_dev_agreement)
+
+    def test_read_dev_agreement_skip(self):
```

Comment:
for readability, I think it would be good to add a comment saying "the logged in user already has a read_dev_agreement timestamp." This is part of why I dislike fixtures -- they hide all the data requirements for each test